package.json

```diff
@@ -14,12 +14,12 @@
     "start": "webpack-dev-server --mode development --config webpack.config.cjs"
   },
   "dependencies": {
+    "@graphy/content.ttl.read": "^4.3.4",
```

Comment:
Nothing against it, but what provides @graphy/content.ttl that @rdf-esm/formats-common doesn't?

Reply:
A parser is only used to load the sample resources. The formats package uses n3, which acts weird when bundled in wc.dev. Strangely, it appears as a `this` inside rdf-ext's `blankNode` which throws because there is an undefined property.

tl;dr; must be caused by the fact that there are duplicates of some of those packages where newer are ESM and older are CJS